### ISSUE TITLE
[coati] why logits and sequences need to shift?

### ISSUE THREAD

**Issue by gongel** [hpcaitech/ColossalAI#3319]
hi, why logits and sequences need to shift?
https://github.com/hpcaitech/ColossalAI/blob/64350029fecb167db415c82ad41cbd8190dde9e5/applications/Chat/coati/models/base/actor.py#L61
thx.

**Comment by gongel** [comment]
Thx. I got it.
